wiki/Gameplay/Game_modifier/Easy_(lazer)/en.md

```diff
@@ -35,7 +35,13 @@ tags:
 *For the osu!stable version of this article, see: [Easy (mod)](/wiki/Gameplay/Game_modifier/Easy)*\
 *For the full list of all [lazer](/wiki/Client/Release_stream/Lazer) mods, see: [Game modifier (lazer)](/wiki/Gameplay/Game_modifier_(lazer))*
 
-<!-- TODO description and settings -->
+The **Easy** mod attempts to make the gameplay on any [beatmap](/wiki/Beatmap) played easier by halving the difficulty settings. In all [game modes](/wiki/Game_mode) except [osu!taiko](/wiki/Game_mode/osu!taiko), the mod allows you to have two extra lives in case your [health bar](/wiki/Client/Interface/Health_bar) drops to the zero, which refills it completely and instantly.
```

Comment:
shouldn't use "you" on the wiki

```suggestion
The **Easy** mod attempts to make the gameplay on any [beatmap](/wiki/Beatmap) played easier by halving each difficulty setting. In all [game modes](/wiki/Game_mode) except [osu!taiko](/wiki/Game_mode/osu!taiko), the mod gives two extra lives in case the [health bar](/wiki/Client/Interface/Health_bar) drops to zero, which instead refills it instantly.
```